### DISCUSSION TITLE
Unable to fetch user details in react using Supabase

### DISCUSSION THREAD

**Discussion by creyo** [supabase/supabase#18075] [Questions]
Hiii ,When fetching data, I encounter the word 'undefined,' which means that the data I'm trying to access is not defined or does not exist.       

const { user, error } = await supabase.auth.signInWithOtp({
        email,
        password,
      });

      if (error) {
        setError(error.message);
        return;
      }

      console.warn( "user", user) // at this point i m getting undefined

**Comment by GaryAustin1** [comment]
I think you asked this on Discord as well.
The format is `const {data,error} = await.supabase.auth.signInWithOtp()`.
Also note that the session will not come back to that spot as the user is going to be redirected to siteUrl in your case.
